packages/backend/package.json

```diff
@@ -42,7 +42,7 @@
     "testcontainers": "^10.5.0",
     "tmp": "^0.2.1",
     "ts-node": "^10.9.2",
-    "typescript": "^4.9.5"
+    "typescript": "^5.3.3"
```

Comment:
do we actually need to specify this across all of the different packages, or is having it in the root package.json enough?